src/components/utils/DropdownOptions/index.tsx

```diff
@@ -23,16 +23,4 @@ export const optionsStyle = ({ menuPosition }: MenuOptions) => (theme: Theme) =>
   box-shadow: ${theme.elevation['02']};
   border-radius: ${rem(4)};
   z-index: 1;
```

Comment:
This should be in a separate issue so we can discuss it further.

Maybe we can abstract all z indexes in one place and have specific levels for each component?